src/GraphQL/loaders/filesMutations.js

```diff
@@ -5,10 +5,9 @@ import * as defaultGraphQLTypes from './defaultGraphQLTypes';
 import logger from '../../logger';
 
 const handleUpload = async (upload, config) => {
-  const data = Buffer.from(await upload.arrayBuffer());
+  const data = await upload.buffer();
```

Comment:
I am not sure if you should rely on an implementation specific `buffer` field which is not part of Fetch spec.

Reply:
using arrayBuffer i got some weird behavior and corrupted file. using buffer worked. the buffer() method seems implemented by @whatwg-node if i remember correctly during my investigation @ardatan.

I got some huge issue migrating ot Yoga V4, it didn't worked easily out of the box. I landed with the current implementation to get file upload working with multi part limits.

Reply:
HI @ardatan , i tried the arrayBuffer method. But we got actually have something strange. Some content of the request seems to leak into the file content. It do not happen with .buffer() file. 
I'm not sure why.

![image](https://github.com/parse-community/parse-server/assets/27959372/e3958cb9-db05-4612-b695-d09b032c2864)


Here a example in our test suite